src/Models/Conditions.php

```
@@ -7,9 +7,9 @@
 use OpenFGA\Schema\CollectionSchemaInterface;
 
 /**
- * @extends AbstractIndexedCollection<Condition>
+ * @extends IndexedCollection<Condition>
  */
-final class Conditions extends AbstractIndexedCollection implements ConditionsInterface
+final class Conditions extends IndexedCollection implements ConditionsInterface
```

PHPStan (2.1.15)

Class OpenFGA\Models\IndexedCollection specifies template type T of interface OpenFGA\Models\IndexedCollectionInterface as OpenFGA\Models\Condition but it's already specified as OpenFGA\Models\ConditionInterface.


(generics.interfaceConflict)